notify.js

```diff
@@ -25,7 +25,7 @@
         return typeof item === 'function';
     }
 
-    function Notify(title, options) {
+    function Notify(title, options, params) {
```

Comment:
'data' is great!
but I prefere to divide options (as standard Notification() options) and user data